miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/util/form/SearchPaneTag.java

```diff
@@ -29,7 +29,7 @@
   protected int doStartTagInternal() throws Exception {
 
     pageContext.getOut().append(String.format(
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') 3rd argument is not used. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3457)